api/src/main/resources/messages.properties

```diff
@@ -631,6 +631,8 @@ reporting.parameter.startDate=Start Date
 reporting.parameter.endDate=End Date
 reporting.parameter.location=Location
 reporting.parameter.locationList=Locations
+reporting.parameter.attributeType=Attribute Type
+reporting.parameter.values=Value
```

Comment:
I'd also point out that person attributes can be more complex than this.  They have a format, which could be of various data types.  The personattributecohortdefinition supports 3 of these natively - String, Location, and Concept.  The UI may need to be smart enough to use appropriate widgets based on the data type of the attribute type, or to limit the widget to only those that are of type String